ntbs-service/DataAccess/NotificationRepository.cs

```diff
@@ -74,17 +74,21 @@ public async Task AddNotificationAsync(Notification notification)
 
         public IQueryable<Notification> GetRecentNotificationsIQueryable()
         {
-            return GetNotificationsWithBasicInformationIQueryable()
+            return _context.Notification
+                .AsNoTracking()
```

Comment:
Was there a reason to add `AsNoTracking` here? Since we're not retrieving any entities in our query, I wouldn't have expected it to make any difference.

Reply:
I think I needed it one point when I was misusing `[NotMapped]` properties, but I fixed that a while ago so this can be safely removed. Thanks!